jackal_base/scripts/calibrate_compass

```diff
@@ -2,7 +2,8 @@
 
 TEMPDIR=$(mktemp -d --tmpdir calibrate_compass.XXXX)
 BAG_FILE=${TEMPDIR}/imu_record.bag
-CAL_FILE=${TEMPDIR}/imu_compass.yaml
+CAL_FILE=${TEMPDIR}/mag_config.yaml
+CAL_FINAL_PATH=${ROS_ETC_DIR}/jackal_base
```

Comment:
The fallback is `/opt/ros/indigo/share/jackal_base/mag_config_default.yaml`. A use running this script will be invited to create `/opt/ros/indigo/etc/jackal_base/mag_config.yaml`. So nothing is being blasted away, unless we run the calibration ourselves once before shipping, which would not be an unreasonable thing to begin doing.

In any case, a more sophisticated future version of the compass calibration script could offer the user a choice where to put the file (base etc dir, overlay etc dir, `~/.ros`, etc). I know `~/.ros` is the default for camera calibration files, but I don't care for sucking stuff out of a home directory by default from a deb package config. Calibration feels like a "local administrator" task, not a "user" task.